source/sync/app-builder/local-to-sync.txt

```diff
@@ -86,30 +82,19 @@ language/platform:
 
 Copy Existing Data
 ------------------
+At this point, your data still only exists in the local database. You need to 
+perform an initial copy before Realm will sync the data. Most of the SDKs provide 
+an easy process for step. For details, see:
 
-.. note:: Alternate SDK Implementations
-
-   Some SDKs support an easier implementation for adding sync to a
-   local-only realm than the steps outlined on this section.
-
-   - :ref:`Open Non-Synced Realm as a Synced Realm - Swift SDK <ios-open-non-synced-as-synced-realm>`
-   - :ref:`Open Existing Realm with Different Configuration - Node.js SDK <node-open-existing-realm-different-configuration>`
-   - :ref:`Open Existing Realm with Different Configuration - React Native SDK <react-native-open-existing-realm-different-configuration>`
-
-None of the data in your local realm automatically transfers to a
-new synced realm [#f1]_. You must copy the data you want to preserve
-to the new realm. The following diagram and steps outline the process for
-copying all of the data from a local realm to a synced realm.
-
-.. [#f1] With the exception of apps migrated from local to synced realms using the easier implementations outlined in "Alternate SDK Implementations".
+- :ref:`Open Non-Synced Realm as a Synced Realm - Swift SDK <ios-open-non-synced-as-synced-realm>`
+- :ref:`Convert Between Non-Synced Realms and Synced Realms - .NET SDK<dotnet-convert-to-sync>`
+- :ref:`Open Existing Realm with Different Configuration - Node.js SDK <node-open-existing-realm-different-configuration>`
+- :ref:`Open Existing Realm with Different Configuration - React Native SDK <react-native-open-existing-realm-different-configuration>`
```

Comment:
[Blocking] The JS SDKs actually don't have good support for migrating a local realm to Flexible Sync. They can do local -> Partition-Based Sync, but not Flexible Sync. We should remove them from this list.

It's a mess. Here's a thread on it if you're interested: https://mongodb.slack.com/archives/C010R3CMP3N/p1689778002169139?thread_ts=1689775340.646159&cid=C010R3CMP3N